xarray/core/computation.py

```diff
@@ -2046,9 +2046,13 @@ def _calc_idxminmax(
         raise ValueError("Must supply 'dim' argument for multidimensional arrays")
 
     if dim not in array.dims:
-        raise KeyError(f'Dimension "{dim}" not in dimension')
+        raise KeyError(
```

Comment:
It's also a bit inconsistent that idxmin/idxmax raise KeyError while others raise ValueError, but maybe that's not worth breaking.